cluster.yml

```diff
@@ -52,13 +52,23 @@
   any_errors_fatal: "{{ any_errors_fatal | default(true) }}"
   roles:
     - { role: kubespray-defaults}
```

Comment:
nit: to be consistent you could drop the curly brackets too

Reply:
not planning to change it everywhere right now